__test__/server/api/people.test.js

```diff
@@ -0,0 +1,482 @@
+/* eslint-disable no-unused-expressions, consistent-return */
+import { r } from "../../../src/server/models/";
+import { getUsersGql } from "../../../src/containers/PeopleList";
+import { GraphQLError } from "graphql/error";
+
+import {
+  setupTest,
+  cleanupTest,
+  createUser,
+  createInvite,
+  createOrganization,
+  createCampaign,
+  createTexter,
+  createContacts,
+  assignTexter,
+  updateUserRoles,
+  runGql
+} from "../../test_helpers";
+
+describe("people", async () => {
+  let testTexterUsers;
+  let testAdminUsers;
+  let testContacts;
+  let testCampaigns;
+  let testInvite;
+  let testOrganization;
+  let organizationId;
+  let variables;
+  let cursor;
+
+  beforeEach(async () => {
+    await setupTest();
+
+    testAdminUsers = [];
+    testAdminUsers.push(
+      await createUser({
```

Comment:
Would there be any value in moving reused test data to a common place to reduce duplication?